components/MobileBottomNav/MobileBottomNav.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { FC, useEffect, useState } from 'react'
+import { useRouter } from 'next/router'
+import Link from 'next/link'
+
+import QuestionMarkIcon from 'assets/icons/svg/question-mark.svg'
+import HomeInActiveIcon from 'assets/icons/svg/nav/home-inactive.svg'
+import HomeActiveIcon from 'assets/icons/svg/nav/home-active.svg'
+import SaveInActiveIcon from 'assets/icons/svg/nav/save-inactive.svg'
+import SaveActiveIcon from 'assets/icons/svg/nav/save-active.svg'
+import TeamInActiveIcon from 'assets/icons/svg/nav/team-inactive.svg'
+import TeamActiveIcon from 'assets/icons/svg/nav/team-active.svg'
+import SearchAciveIcon from 'assets/icons/svg/nav/search-active.svg'
+import SearchInAciveIcon from 'assets/icons/svg/nav/search-inactive.svg'
+
+const MobileBottomNav: FC = () => {
+  const router = useRouter()
+
+  const inActiveIconCls = 'stroke-gray-400'
+  const activeIconCls = 'fill-primary dark:fill-white'
+
+  const navLinks = [
+    {
+      inActiveIcon: <HomeInActiveIcon className={inActiveIconCls} />,
+      activeIcon: <HomeActiveIcon className={activeIconCls} />,
+      label: 'Home',
+      href: '/',
+      spaceActive: true,
+    },
+    {
+      inActiveIcon: <SaveInActiveIcon className={inActiveIconCls} />,
+      activeIcon: <SaveActiveIcon className={activeIconCls} />,
+      label: 'Saved',
+      href: '/saved',
+    },
+    {
+      inActiveIcon: <SearchInAciveIcon className={inActiveIconCls} />,
+      activeIcon: <SearchAciveIcon className={activeIconCls} />,
+      label: 'Search',
```

Comment:
We need to pass `onClick` function from here. 

so we can do:
```js
onClick 
   ? <button onClick={onClick} className={commonCls}>{...}</button>
   : <Link href={href} className={commonCls}>{...}</Link>
```

This approach is much better and simpler. What do you think?